src/amo/components/GuidesAddonCard/index.js

```diff
@@ -39,30 +40,44 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
   render() {
     const { addon, i18n, staffPick } = this.props;
 
-    return addon ? (
+    return addon !== null ? (
       <Card>
         <AddonInstallError error={this.props.installError} />
         <div className="GuidesAddonCard">
-          <AddonCompatibilityError addon={addon} />
+          {addon && <AddonCompatibilityError addon={addon} />}
```

Comment:
Maybe you could add an assertion for that too (in the newly added test case)

Reply:
I have a test for AddonCompatabilityError already? Do you mean show that is has no length when there is no addon?

Reply:
yep

Reply:
got it - thanks!